_universum/modules/launcher.py

```diff
@@ -235,6 +304,20 @@ def define_arguments(argument_parser):
         parser.add_argument("--launcher-config-path", "-lcp", dest="config_path", metavar="CONFIG_PATH",
                             help="Project configs.py file location. Mandatory parameter")
 
+        parser.add_argument("--run-step", "-run", dest="run_step",
```

Comment:
If this is a launcher parameter, it should start with '--launcher-'; also I'm not sure if 'run step' is clear enough. Could you test this parameter name on someone